app/src/main/java/com/cornellappdev/transit/models/Place.kt

```diff
@@ -12,24 +13,22 @@ data class QueryResult(
     @Json(name = "busStops") var stops : List<Place>
 )
 
-//TODO: Deprecated, start migrating to Place data class
 /**
- * DEPRECATED: Data class representing a TCAT stop
+ * Enum class representing the types of TCAT places
  */
-data class Stop(
-    @Json(name = "lat") var latitude: Double,
-    @Json(name = "long") var longitude: Double,
-    @Json(name = "name") var name: String,
-    @Json(name = "type") var type: String,
-)
+enum class Type() {
+    busStop,
+    applePlace
```

Comment:
plz name any enums as `BUS_STOP` convention :)

Reply:
I did it like this because in the response from the backend, the possible values for the `type` field are these. If I have to change it to the `BUS_STOP` convention I think I should be adding a type adapter to Moshi (?). Is that alright?